### PR TITLE
Added Radio player to Nitro !

### PR DESCRIPTION
**Pull request by duckietm**
- Update your UI-CONFIG.JSON to the right stream or ofcourse you can use the default
- If you do not want this, just set enable.radioplayer to false

And feel free to customize it as much as you want !

Have funn, and share with someone !

image

### CODE REVIEW

File: apps/frontend/src/components/right-side/RightSideView.tsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ export const RightSideView: FC<{}> = props => {
         <OfferView />
         <RoomPromotesWidgetView />
         <NotificationCenterView />
+        <RadioView />
```
Comment (Walidoux) [inline apps/frontend/src/components/right-side/RightSideView.tsx:21]: Unmount the component from the DOM if the user has disabled the radio from the UI CONFIG

File: apps/frontend/src/components/right-side/radioview/RadioView.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import {useState} from "react";
+import {FaPlayCircle, FaPauseCircle} from "react-icons/fa";
+import {GetConfiguration} from "../../../api";
+import {Base, Text, Column, Flex, Button, LayoutGridItem} from "../../../common";
+
+export const RadioView = () => {
+  const [audio, setAudio] = useState(new Audio(GetConfiguration<string>("radio.stream.url")));
+  const [isPlaying] = useState(false);
```
Comment (Walidoux) [inline apps/frontend/src/components/right-side/radioview/RadioView.tsx:8]: - Unused state and unused events down here.
- You don't have to create handlers for pausing and starting the radio. Keep track of the status radio with `paused` property as mentioned in MDN docs: 
image
and for changing the styles you can do something like that:

```tsx
<Button  
    onClick={() => audio.play()}
    type="button"
    style={{
          backgroundColor: "#F1B941",
          color: audio.paused ? "black" : "blue",
          cursor: "pointer",
     }}
>
```

so as the same for the button which handles the pause event

**Review by Walidoux — COMMENTED** [review]
WIP, still needs enhancements

**Review by Walidoux — COMMENTED** [review]
LGTM

**Review by Walidoux — APPROVED** [review]
LGTM

**Comment by duckietm** [comment]
Better to use CMS for this

**Pull request closed without merging**